[IOTDB-2313][REST API Doc] how to avoid GC overhead (#4926)
apache/iotdb: docs/UserGuide/Ecosystem Integration/Grafana Plugin.md

*  `group by ([0, 20), 2ms, 3ms), level=1`

> Tip: to avoid oom problems, select * from root is not recommended xx.** This statement is used in the grafana plugin.

Reviewer: ```suggestion
Tip: Statements link `select * from root.xx.**` are not recommended because those statements may cause OOM.
```
